.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: >-
+      ruby-${{ matrix.ruby }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.6', '2.7' ]
+
+    steps:
+      - name: repo checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby ${{ matrix.ruby }}
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+
+      - name: bundle install
+        run: |
+          ./script/bootstrap
+
+      - name: test
+        run: ./script/ci_build
```

Comment:
This series of steps is what TravisCI does for us now: checks out the code, installs the dependencies, and runs the tests.